src/generated/stdout/quic_trace.c

```diff
@@ -19,9 +19,9 @@
 #include <quic_trace.h>
 
 
-void EncodeHexBuffer(_In_reads_(BufferLen) uint8_t * Buffer,
```

Comment:
I rather ignore this error than make this change. This code was specifically designed for small buffers where size fit in `uint8_t`.